stdeb/util.py

```diff
@@ -1116,6 +1116,14 @@ def __init__(self,
 
         if not (with_python2 or with_python3):
             raise RuntimeError('nothing to do - neither Python 2 or 3.')
+
+        if with_python2:
+            if shutil.which("python"):
+                self.python2_binname = "python"
+            elif shutil.which("python2"):
+                self.python2_binname = "python2"
+            else:
+                raise RuntimeError("Python 2 binary not found on path as either `python` or `python2`")
```

Comment:
shutil.which isn't actually available in python2 and none of the preliminary testing exposed this.